### PR TITLE
Add tests to OrbitAPIValidator

### PR DESCRIPTION
**Pull request by kirkbyo**
Add a few test cases to the OrbitAPIValidator introduced in https://github.com/andymatuschak/orbit/pull/208. It required setting up `jest` for the `api` package.

### CODE REVIEW

File: packages/api/jest.config.cjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  testEnvironment: "node",
+  testMatch: ["**/?(*.)+(spec|test).ts?(x)"],
+  testPathIgnorePatterns: ["dist", "node_modules"],
+  transformIgnorePatterns: [],
+  transform: {
+    "\\.[jt]s$": ["babel-jest", { cwd: __dirname }],
+  },
+  // Jest struggles to read these mappings from the multiformats packages, which seem to be specifying them just fine as far as I can tell.
+  moduleNameMapper: {
```
Comment (andymatuschak) [inline packages/api/jest.config.cjs:10]: :roll_eyes: Such a shame we have to repeat this in all our packages' configs.

**Review by andymatuschak — APPROVED** [review]
Looks great. Could I trouble you (either in this PR or a new one) for a test exercising the type coercion semantics we expect?

File: packages/api/jest.config.cjs (hunk 1 above)
Comment (kirkbyo) [reply packages/api/jest.config.cjs:10]: Is there any particular reason why we can't set this up at the packages root? Is the difference in babel configs the blocker?

**Comment by kirkbyo** [comment]
Done, good call!

File: packages/api/jest.config.cjs (hunk 1 above)
Comment (andymatuschak) [reply packages/api/jest.config.cjs:10]: Maybe we can! I'm not sure how configuration "inheritance" works for Jest.

**Pull request merged**